app/models/ldap_auth_source.rb

```diff
@@ -54,16 +54,17 @@ def self.unique_attribute
   validates :tls_mode, inclusion: { in: tls_modes.keys }
 
   validates :host, :port, :attr_login, presence: true
-  validates :name, :host, length: { maximum: 60, allow_nil: true }
+  validates :host, length: { maximum: 60, allow_nil: true }
```

Comment:
removal of the length validation for name was intended here?

Reply:
There is already a length validation defined for `name`.

Reply:
I think there is another validation that is also declared twice (I need to double-check still, hence still WIP!)